addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2110,6 +2110,16 @@ var SnippetsMenu = Widget.extend({
         // drop code need to use the mutex themselves.
         await this.postSnippetDropPromise;
 
+        // TODO remove me in master. This was added as a fix in stable to remove
+        // the "data-snippet" attribute that was added on the "span" element of
+        // the "Cover" snippet when modifying the "Parallax" of the snippet.
+        window.document.querySelectorAll("span[data-snippet='s_cover'][data-name='Cover']")
+                    .forEach(el => {
+            delete el.dataset["snippet"];
+            delete el.dataset["name"];
+            const dirty = el.closest(".o_editable") || el;
+            dirty.classList.add("o_dirty");
+        });
```

Comment:
@qsm-odoo , we were wondering with @bso-odoo if you would prefer to let those lines of code in master (as some website pages might not have been edited so those lines might not have been be executed) or rather remove those lines and go for a migration script ?

Reply:
@loco-odoo Migration script in this case :+1: